<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@920.v59f71ce16f04">
  <node class="cps.n.StepEndNode" plugin="workflow-cps@3908.vd6b_b_5a_a_54010">
    <parentIds>
      <string>41</string>
    </parentIds>
    <id>42</id>
    <startId>11</startId>
  </node>
  <actions>
    <wf.a.TimingAction plugin="workflow-api@1322.v857eeeea_9902">
      <startTime>1721546642292</startTime>
    </wf.a.TimingAction>
    <wf.a.ErrorAction plugin="workflow-api@1322.v857eeeea_9902">
      <error class="hudson.AbortException">
        <detailMessage>script returned exit code 127</detailMessage>
        <stackTrace>
          <trace>PluginClassLoader for workflow-durable-task-step//org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.handleExit(DurableTaskStep.java:668)</trace>
          <trace>PluginClassLoader for workflow-durable-task-step//org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.check(DurableTaskStep.java:614)</trace>
          <trace>PluginClassLoader for workflow-durable-task-step//org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.run(DurableTaskStep.java:555)</trace>
          <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)</trace>
          <trace>java.base/java.util.concurrent.FutureTask.run(Unknown Source)</trace>
          <trace>java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)</trace>
          <trace>java.base/java.lang.Thread.run(Unknown Source)</trace>
        </stackTrace>
        <suppressedExceptions>
          <wf.a.ErrorAction_-ErrorId>
            <stackTrace/>
            <suppressedExceptions class="empty-list"/>
            <uuid>465fa695-84d1-47ae-831d-565d14fac61a</uuid>
          </wf.a.ErrorAction_-ErrorId>
        </suppressedExceptions>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>